iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/executable/ExecutableManager.java

```diff
@@ -130,6 +130,31 @@ public boolean hasFileUnderInstallDir(String fileName) {
         Paths.get(this.libRoot + File.separator + INSTALL_DIR + File.separator + fileName));
   }
 
+  public boolean hasPluginFileUnderInstallDir(String pluginName, String fileName) {
```

Comment:
Better place those in PipePluginExecutableManager first. UDF and Trigger may not use them because they may have large jar package and require high reusability....

Reply:
Modification completed